src/components/RadioButtonLarge/__tests__/RadioButtonLarge.spec.js

```diff
@@ -0,0 +1,82 @@
+import '@testing-library/jest-dom';
+import { render, fireEvent } from '@testing-library/vue';
+import RadioButton from '../RadioButtonLarge.vue';
+
+const initialProps = {
+  id: 'test',
+  modelValue: '',
+  name: 'test name',
+  label: 'Test',
+  value: 'test',
+  error: false,
+  disabled: false
+};
+
+describe('Radio Button', () => {
+
+  it('does not check the input when the value does not match the modelValue', () => {
+    const props = initialProps;
+    const { getByLabelText } = render(RadioButton, {
+      props
+    });
+
+    const radio = getByLabelText(props.label);
+    expect(radio).not.toBeChecked();
+  });
+
+  it('checks the input when the value matches the modelValue', () => {
+    const props = {
+      ...initialProps,
+      modelValue: initialProps.value
+    };
+    const { getByLabelText } = render(RadioButton, {
+      props
+    });
+
+    const radio = getByLabelText(props.label);
+    expect(radio).toBeChecked();
+  });
+
+  it('disables the input when disabled prop is true', () => {
+    const props = {
+      ...initialProps,
+      disabled: true
+    };
+
+    const { getByLabelText } = render(RadioButton, {
+      props
+    });
+
+    const radio = getByLabelText(props.label);
+    expect(radio).toBeDisabled();
+  });
+
+  it('adds an error class to the input when error prop is true', () => {
+    const props = {
+      ...initialProps,
+      error: true
+    };
+
+    const { getByLabelText } = render(RadioButton, {
+      props
+    });
+
+    const radio = getByLabelText(props.label);
+    expect(radio).toHaveClass('radio__input--error');
+  });
+
+  it('fires the input and click events when the input is clicked', async () => {
+    const props = initialProps;
+    const { getByLabelText, emitted } = render(RadioButton, {
+      props
+    });
+    const radio = getByLabelText(props.label);
+
+    await fireEvent.click(radio);
```

Comment:
`fireEvent` often clicks on things when they're in a non-clickable state (which `userEvent` does not do, rightfully) - can we check to make sure the button isn't disabled/hidden/etc. in this test?

Reply:
Ok, here is my theory on this: fireEvent is working because we are grabbing this input by label and the actual input is 'hidden' behind our psuedo/custom component... - so userEvent is not working because it is, in fact, hidden

Reply:
I was importing userEvent incorrectly - problem solved!